### PR TITLE
Add Ord instance for Address

### PR DESCRIPTION
**Pull request by klntsky**

### CODE REVIEW

File: src/Serialization/Address.purs [hunk 1]
```diff
@@ -168,6 +168,9 @@ showVia nm toShowable addr = "(" <> nm <> " " <> show (toShowable addr) <> ")"
 instance Eq Address where
   eq = eq `on` addressBytes
 
+instance Ord Address where
+  compare a b = addressBytes a `compare` addressBytes b
```
Comment (vvtran) [inline src/Serialization/Address.purs:172]: Just to follow a common convention in our libraries, I think `compare = compare on addressBytes` would be better, and it's point free.

**Review by vvtran — COMMENTED** [review]
One comment.

**Comment by vvtran** [comment]
I think this PR is redundant now, as this change was included in https://github.com/Plutonomicon/cardano-browser-tx/pull/177 which has since been merged.

**Comment by ngua** [comment]
Made redundant by #177

**Pull request closed without merging**